lms/djangoapps/instructor/tests/views/test_instructor_dashboard.py

```diff
@@ -103,6 +103,21 @@ def has_instructor_tab(user, course):
         student = UserFactory.create()
         self.assertFalse(has_instructor_tab(student, self.course))
 
+    def test_student_admin_staff_instructor(self):
+        """
+        Verify that staff users are not able to see course-wide options, while still
+        seeing individual learner options.
+        """
+        # Original (instructor) user can see the course-wide grade adjustment
+        response = self.client.get(self.url)
+        self.assertIn('<h4 class="hd hd-4">Adjust all enrolled learners', response.content)
+
+        # But staff user cannot
+        staff = StaffFactory(course_key=self.course.id)
+        self.client.login(username=staff.username, password="test")
+        response = self.client.get(self.url)
+        self.assertNotIn('<h4 class="hd hd-4">Adjust all enrolled learners', response.content)
```

Comment:
Is there a test that makes sure the staff user can see the "View" section?  That was the original bug.